lib/libc/math/Make.defs

```diff
@@ -81,9 +81,14 @@ CSRCS += lib_tanl.c lib_tanhl.c lib_asinhl.c lib_acoshl.c lib_atanhl.c lib_erfl.
 CSRCS += lib_copysignl.c lib_truncl.c lib_nextafterl.c lib_nexttowardl.c lib_remainderl.c
 CSRCS += lib_remquol.c
 
+CSRCS += lib_libexpi.c lib_libsqrtapprox.c
```

Comment:
Actually this line is not modified, but because of location change, it looks it gets modified. If there is no specific reason to move, how about keep this as previous?